snomed/com.b2international.snowowl.snomed.datastore/src/com/b2international/snowowl/snomed/datastore/index/entry/SnomedOWLRelationshipDocument.java

```diff
@@ -68,6 +71,21 @@ public static SnomedOWLRelationshipDocument createFrom(final SnomedRelationship
 			return create(r.getTypeId(), r.getDestinationId(), r.getRelationshipGroup());
 		}
 	}
+	
+	public static SnomedOWLRelationshipDocument createFrom(final SnomedOWLRelationship r) {
+		if (r.hasValue()) {
+			return createValue(r.getTypeId(), r.getValueAsObject(), r.getRelationshipGroup());
+		} else {
+			return create(r.getTypeId(), r.getDestinationId(), r.getRelationshipGroup());
+		}
+	}
+	
+	public static List<SnomedOWLRelationshipDocument> createFrom(List<SnomedOWLRelationship> owLRelationships) {
+		if (owLRelationships == null) {
+			return null;
+		}
+		return owLRelationships.stream().map(SnomedOWLRelationshipDocument::createFrom).toList();
```

Comment:
`toList()` returns an unmodifiable list backed by an array. Is it acceptable for callers?

Reply:
From the perspective of this method, yes. If a caller would like to modify this list it needs to wrap the contents in a mutable array and modify that.